gnat2goto/driver/goto_utils.adb

```diff
@@ -120,6 +121,22 @@ package body GOTO_Utils is
       return Ret;
    end Symbol_Expr;
 
+   procedure New_Constant_Symbol_Entry (N : Node_Id;
```

Comment:
If the `Node_Id` is just used to get a name and a source location, can we just pass those to the function instead? The function doesn't seem node specific.

Reply:
Yes, `node_id` is used for the same purpose as in the original procedure.